configuration/appointmentsservicesdefinitions/appointmentsservicesdefinitions.csv

```diff
@@ -0,0 +1,7 @@
+Uuid,Void/Retire,Name,Description,Duration,Start Time,End Time,Max Load,Speciality,Location,Label Colour,_order:1001
+a6cf2d24-56b9-4adc-ba8b-f37307b5c6a5,,Prenatal,,,,,,250062fa-5bf8-446a-a49d-94f6679d5904,Haiti,#00FF00,
+55c7fb92-b5b1-4d2d-a0f9-abeeb8552f2b,,Family Planning,,,,,,250062fa-5bf8-446a-a49d-94f6679d5904,Haiti,#0000FF,
+c8c14bd2-1d7b-4822-bf12-77c415b8dbef,,Infant vaccination,,,,,,14106bb7-dad6-4446-809d-737f4c128ae3,Haiti,#33fff9,
+be6c3bea-f7d6-4686-9de3-ee799d68a63f,,Malnutrition,,,,,,14106bb7-dad6-4446-809d-737f4c128ae3,Haiti,#FFFF00,
+09c85fcf-a43e-424a-b3e3-8671b09eda5d,,General,,,,,,9f2a8cd0-32c6-4844-8df7-1ac9c4d79943,Haiti,#b533ff,
+ef11cdc5-c96d-43c9-8229-cb9c75f02256,,TB,,,,,,592cea65-7bac-45ab-8cc5-7a7a90af3121,Haiti,#ec33ff,
```

Comment:
Let's try to find a more contrasting color for this one (that's too similar to General)